pom.xml

```diff
@@ -46,7 +46,8 @@
         <version.kafka>3.2.1</version.kafka>
         <version.org.slf4j>1.7.36</version.org.slf4j>
         <version.logback>1.4.0</version.logback>
-        <version.ybclient>0.8.21-20220715.193438-1</version.ybclient>
+        <!-- <version.ybclient>0.8.21-20220715.193438-1</version.ybclient> -->
+        <version.ybclient>0.8.28-SNAPSHOT</version.ybclient>
```

Comment:
Did you compile locally? The version should point to one with timestamp and not SNAPSHOT.

Reply:
Yes, because at the time of compilation, the upstream one was not present, will change it now.